module3/homework/inner_product/arithmeticAverage.cpp

```diff
@@ -0,0 +1,16 @@
+#include <math.h>
+#include <functional>
+#include <numeric>
+#include <vector>
+
```

Comment:
Sorry for being boring xd I think that You will not modify iterators in this exercise. Of course it isn't any issue. It's just good practice in my opinion. I suggest you to use constant iterators whenever you won't modify them.

Reply:
Again, good point!